qjs.c

```diff
@@ -441,16 +438,16 @@ int main(int argc, char **argv)
     while (optind < argc && *argv[optind] == '-') {
         char *arg = argv[optind] + 1;
         const char *longopt = "";
-        char *opt_arg = NULL;
+        char *optarg = NULL;
```

Comment:
Renamed for consistency (if you call optind optind, then call optarg optarg, amirite?)